### PR TITLE
test: jestify v21.4.1 invoke contract test

### PR DESCRIPTION
**Pull request by Leeyoungone**
Migrate Tap to Jest

File Path:
packages/cactus-plugin-ledger-connector-quorum/
src/test/typescript/
integration/plugin-ledger-connector-quorum/
deploy-contract/
v21.4.1-invoke-contract.test.ts

This is a PARTIAL resolution to issue #238

Signed-off-by: Youngone Lee <youngone.lee@accenture.com>

### CODE REVIEW

**Review by takeutak — APPROVED** [review]

File: packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts [hunk 1]
```diff
@@ -24,182 +24,196 @@ import { PluginRegistry } from "@hyperledger/cactus-core";
...
 const contractName = "HelloWorld";
+const testCase = "Quorum Ledger Connector Plugin";
 
-test("Quorum Ledger Connector Plugin", async (t: Test) => {
-  const containerImageVersion = "2021-05-03-quorum-v21.4.1";
-  const ledgerOptions = { containerImageVersion };
-  const ledger = new QuorumTestLedger(ledgerOptions);
-  test.onFinish(async () => {
+describe(testCase, () => {
+  // test("Quorum Ledger Connector Plugin", async (t: Test) => {
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts:30]: @Leeyoungone No commented code unless you need it for something specific please :-)

File: packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts [hunk 2]
```diff
@@ -24,182 +24,196 @@ import { PluginRegistry } from "@hyperledger/cactus-core";
...
+      },
     });
-    t2.ok(setNameOut, "setName() invocation #1 output is truthy OK");
 
-    try {
-      const setNameOutInvalid = await connector.getContractInfoKeychain({
+    const balance = await web3.eth.getBalance(testEthAccount.address);
+    expect(balance).toBeTruthy();
+    expect(parseInt(balance, 10)).toEqual(10e9);
+    // t.equals(parseInt(balance, 10), 10e9, "Balance of test account is OK");
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts:102]: @Leeyoungone No commented code unless you need it for something specific please :-)

File: packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts [hunk 3]
```diff
@@ -24,182 +24,196 @@ import { PluginRegistry } from "@hyperledger/cactus-core";
...
+
+    let contractAddress: string;
+
+    {
+      // test("deploys contract via .json file", async (t2: Test) => {
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts:107]: @Leeyoungone No commented code unless you need it for something specific please :-)

File: packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts [hunk 4]
```diff
@@ -24,182 +24,196 @@ import { PluginRegistry } from "@hyperledger/cactus-core";
...
+        web3SigningCredential: {
+          ethAccount: firstHighNetWorthAccount,
+          secret: "",
+          type: Web3SigningCredentialType.GethKeychainPassword,
+        },
+      });
+      expect(helloMsg).toBeTruthy();
+      // t2.ok(helloMsg, "sayHello() output is truthy");
+      expect(typeof helloMsg === "string").toBeTrue();
+      // t2.true(
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts:144]: @Leeyoungone No commented code unless you need it for something specific please :-)

File: packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts [hunk 5]
```diff
@@ -24,182 +24,196 @@ import { PluginRegistry } from "@hyperledger/cactus-core";
...
+      //   typeof helloMsg === "string",
+      //   "sayHello() output is type of string",
+      // );
+      // });
+    }
+
+    // test("invoke Web3SigningCredentialType.GETHKEYCHAINPASSWORD", async (t2: Test) => {
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts:151]: @Leeyoungone No commented code unless you need it for something specific please :-)

**Review by petermetz — CHANGES_REQUESTED** [review]

File: packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts [hunk 6]
```diff
@@ -24,182 +24,196 @@ import { PluginRegistry } from "@hyperledger/cactus-core";
...
+          gas: 1000000,
+          web3SigningCredential: {
+            ethAccount: firstHighNetWorthAccount,
+            secret: "",
+            type: Web3SigningCredentialType.GethKeychainPassword,
+          },
+          nonce: 2,
+        });
+        expect(setNameOutInvalid.transactionReceipt).toBeFalsy();
+        // t2.ifError(setNameOutInvalid.transactionReceipt);
```
[the remark this answers is not in the document]
Comment (Leeyoungone) [reply packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts]: got it! so sorry for the extra effort you had to put into this !!

[the remark this answers is not in the document]
Comment (petermetz) [reply packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts]: > got it! so sorry for the extra effort you had to put into this !!

@Leeyoungone No worries, thank you for diligently addressing all the nit picks of mine. :-)

**Review by petermetz — APPROVED** [review]
@Leeyoungone LGTM, thank you!

**Pull request merged**